lib/scenic/adapters/postgres.rb

```diff
@@ -166,6 +170,15 @@ def try_index_create(index)
         connection.execute("ROLLBACK TO SAVEPOINT #{index.index_name}")
         "index '#{index.index_name}' on '#{index.object_name}' is no longer valid and has been dropped."
       end
+
+      def version_number
+        @version_number ||= connection.select_value('select version()')
+          .match(/PostgreSQL (\d.\d.\d)/).captures[0].tr('.', '').to_i
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.